bin/packages/build.js

```diff
@@ -48,7 +48,7 @@ function getPackageName( file ) {
 }
 
 const isJsFile = ( filepath ) => {
-	return /.\.js$/.test( filepath );
+	return /.\.(js|ts|tsx)$/.test( filepath );
```

Comment:
These bin changes are cherry picked from another PR branch to get the package build working properly. I can discard them before merge.